media-sound/drumkv1/drumkv1-9999.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit fdo-mime
+
+DESCRIPTION="An old-school all-digital drum-kit sampler synthesizer with stereo fx"
+HOMEPAGE="http://drumkv1.sourceforge.net/"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3 autotools
+	EGIT_REPO_URI="https://github.com/rncbc/${PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://download.sourceforge.net/${PN}/${P}.tar.gz"
```

Comment:
I don't think it makes a huge difference, but I'm inclined to go with SourceForge because those seem to be the intended "final" tarballs. If we find in the future that e.g. upstream starts neglecting to release the SF tarballs in a timely fashion even though releases are tagged on GitHub, we could switch.